scripts/update-tezos.sh

```diff
@@ -24,13 +24,11 @@ source scripts/version.sh
 cd ..
 rm -rf upstream-repo
 
-git clone --bare https://gitlab.com/tezos/opam-repository.git upstream-opam-repository
-opam_repository_branch="$(git --git-dir=upstream-opam-repository for-each-ref --contains "$opam_repository_tag" --count=1 --sort='-refname' --format="%(refname:short)" refs/)"
-rm -rf upstream-opam-repository
-
 branch_name="auto/$latest_upstream_tag-release"
 
-our_tezos_tag="$(jq -r '.tezos.ref' nix/nix/sources.json | cut -d'/' -f3)"
+our_tezos_tag="$(jq -r '.tezos_ref' meta.json | cut -d'/' -f3)"
+
+jq ".tezos_ref=$latest_upstream_tag" meta.json | tee meta.json > /dev/null
```

Comment:
Failed for me locally, seems like quotes are missing
```suggestion
jq ".tezos_ref=\"$latest_upstream_tag\"" meta.json | tee meta.json > /dev/null
```